### PR TITLE
I have replaced alll the as const objects with string enums

### LINKED ISSUE #1768: Use enums instead of `as const` objects
This issue is to replace all occurrences of `as const` objects with string enums.

Example:
```ts
export const TOKENS_MODAL_TYPES = {
  activate: 'activate',
  deactivate: 'deactivate',
} as const;
```

should become:
```ts
export enum TokensModalType {
  Activate = 'activate',
  Deactivate = 'deactivate'
}
```

### PR DESCRIPTION
**Pull request by malcyb123**
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

Resolves #1768

### CODE REVIEW

File: src/components/v5/common/ActionSidebar/partials/consts.tsx [hunk 1]
```diff
@@ -5,11 +5,11 @@ import { ACTION } from '~constants/actions';
 
 import { CardSelectOption } from '../../Fields/CardSelect/types';
 
-export const DISTRIBUTION_METHOD = {
-  Equal: 'equal',
-  Unequal: 'unequal',
-  ReputationPercentage: 'reputation-percentage',
-} as const;
+export enum DISTRIBUTION_METHOD {
```
Comment (jakubcolony) [inline src/components/v5/common/ActionSidebar/partials/consts.tsx:8]: Please could you ensure the casing in the names of enums and their members is exactly as in the issue description? This enum should become `DistributionMethod`

File: src/components/v5/common/ActionSidebar/partials/consts.tsx [hunk 2]
```diff
@@ -5,11 +5,11 @@ import { ACTION } from '~constants/actions';
...
+  Equal = 'equal',
+  Unequal = 'unequal',
+  ReputationPercentage = 'reputation-percentage',
+}
 
 export type DistributionMethod =
   (typeof DISTRIBUTION_METHOD)[keyof typeof DISTRIBUTION_METHOD];
```
Comment (jakubcolony) [inline src/components/v5/common/ActionSidebar/partials/consts.tsx:15]: This is wholly on me for not being specific enough in the issue but the idea behind switching to `enum`s was that we can ditch those types. Could you replace those types with newly added enums everywhere?

**Review by jakubcolony — CHANGES_REQUESTED** [review]
Hi @malcyb123, thanks for your first PR, your help is much appreciated!

Your PR looks good, I just left you two comments before I can approve. Thanks

**Comment by chmanie** [comment]
Thank you so much for your work! I just have one little comment and that's on the PR description. We'd really like you to use the template and for you to write a very brief sentence about what you changed. Thanks again ❤️

File: package.json [hunk 3]
```diff
@@ -71,7 +71,7 @@
     "@types/cleave.js": "^1.4.7",
     "@types/dompurify": "^3.0.5",
     "@types/enzyme": "^3.10.3",
-    "@types/jest": "^26.0.0",
+    "@types/jest": "^26.0.24",
```
Comment (rdig) [inline package.json:74]: Please revert the package updates, as they are not relevant to this PR

File: package.json [hunk 4]
```diff
@@ -257,7 +257,7 @@
     "scroll-into-view-if-needed": "^2.2.16",
     "siwe": "^2.1.4",
     "use-local-storage": "^3.0.0",
-    "uuid": "^9.0.0",
+    "uuid": "^9.0.1",
```
Comment (rdig) [inline package.json:260]: Please revert the package updates, as they are not relevant to this PR

**Review by rdig — CHANGES_REQUESTED** [review]
Mostly good, but please revert the package updates first.

After that, I'll approve it.

**Comment by jakubcolony** [comment]
Hi @malcyb123, are you still interested in finishing this PR? Thanks.

**Comment by malcyb123** [comment]
> Hi @malcyb123, are you still interested in finishing this PR? Thanks.

Hello, I am sorry I don't think i will be able to complete it due to some emergencies. Thanks

**Comment by rdig** [comment]
Closing this as no longer having any activity. I've also unassigned #1768 so it's now up for grabs again

**Pull request closed without merging**